cmd/tailscale/cli/debug.go

```diff
@@ -166,6 +166,16 @@ var debugCmd = &ffcli.Command{
 				return fs
 			})(),
 		},
+		{
+			Name:      "set-expire",
+			Exec:      runSetExpire,
+			ShortHelp: "manipulate node key expiry for testing",
+			FlagSet: (func() *flag.FlagSet {
+				fs := newFlagSet("set-expire")
+				fs.DurationVar(&setExpireArgs.in, "in", 0, "if non-zero, set node key to expire this duration from now")
```

Comment:
Can we indicate that duration is seconds and not ms/h/d?

Reply:
those all work. It's a duration flag var, so it requires a unit suffix.

Reply:
👍